lib/philomena_web/controllers/api/json/forums_controller.ex

```diff
@@ -0,0 +1,41 @@
+defmodule PhilomenaWeb.Api.Json.ForumsController do
+  use PhilomenaWeb, :controller
+
+  alias PhilomenaWeb.ForumJson
+  alias Philomena.Forums.Forum
+  alias Philomena.Repo
+  import Ecto.Query
+
+  def show(conn, %{"id" => id}) do
+    forum = 
+      Forum
+      |> where(id: ^id)
+      |> Repo.one()
+
+    cond do
+      is_nil(forum) ->
+        conn
+        |> put_status(:not_found)
+        |> text("")
+
+      forum.access_level != "normal" ->
```

Comment:
You can fold this condition into the above query by just specifying the condition that the forum's `access_level` has to be `normal`.